back/engines/free/email_campaigns/app/models/email_campaigns/campaigns/user_digest.rb

```diff
@@ -104,7 +104,9 @@ def generate_commands(recipient:, time: nil)
     def content_worth_sending?(_)
       # Check positive? as fetching a non-integer env var would result in zero and this hook would return true,
       # whilst top_ideas would be limited to zero ideas, possibly resulting in no content being sent.
-      @content_worth_sending ||= trending_ideas.size >= N_TOP_IDEAS && N_TOP_IDEAS.positive?
+      @content_worth_sending ||=
+        (recent_ideas.size >= N_TOP_IDEAS && N_TOP_IDEAS.positive?) ||
```

Comment:
If `N_TOP_IDEAS.positive?` looks confusing to you too, I guess we can remove it after https://github.com/CitizenLabDotCo/citizenlab/commit/89c3241#diff-e22282ab9112ca0131712c795cc0b92351e590c9a1a5f3a371e29e16929c9001L42
Up to you of course.